diff --git a/CMakeLists.txt b/CMakeLists.txt
index 624960e1a..4a419268a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -35,7 +35,7 @@ find_package(Boost REQUIRED)
 if(Boost_VERSION LESS 106700)
   set(BOOST_PYTHON_COMPONENT python)
 else()
-  set(BOOST_PYTHON_COMPONENT python${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR})
+  set(BOOST_PYTHON_COMPONENT python37)
 endif()
 
 find_package(Boost REQUIRED COMPONENTS
